crates/bevy_transform/src/components/transform.rs

```diff
@@ -207,6 +223,24 @@ impl Transform {
         value
     }
 
+    /// Returns the result of applying this [`Transform`] to a [`Vec3`] interpreted as a point.
+    ///
+    /// This applies rotation, scale, and translation. It's the equivalent of using w=1 in
+    /// homogeneous coordinates.
+    #[inline]
+    pub fn transform_point(&self, point: Vec3) -> Vec3 {
```

Comment:
Does having two functions of the same name but different function arguments work in Rust?

Reply:
rust doesn't have function overloading, so it should not work.

Reply:
One of the functions is on `Transform`, the other on `GlobalTransform`.